docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. JavaScript provides benefits like :- 
+* JavaScript in the browser can reduce server traffic by validating user input before it is sent to the server.
+* Immediate feedback to the site visitors so that they don't have to reload page just to get error messages on form validations.
+* Increased interactivity and richer interfaces allowing content changes on mouse hover, drag and drop gestures, and animations.
+
+Client-side JavaScript is interpreted in the browser without requiring compilation. This allows interactive content to be included in HTML pages which would be static otherwise.
+
+Server-Side JavaScript(Node.js) is JavaScript that enables back-end access to databases, file systems, and servers. NodeJS is built on Google Chrome's JavaScript V8 Engine. NodeJS uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node can be a great solution for applications requiring I/O bound operations, data streaming etc. More details can be found  [here](https://nodejs.org/en/about/)
+
+You should learn JavaScript because:- 
+* It's easy to learn.
+* Its versatile in the sense that it's multi-paradigm supporting including procedural, event based, object oriented and functional programming.
```

Comment:
The first "Its" should be "It's".  The second one should be "its" rather than "it's".